src/components/app-configure/store/StorageRegion.vue

```diff
@@ -30,25 +23,17 @@ function handleChange(option: Region) {
         ></template
       >
       <VStack direction="column" gap="0.75rem" align="start">
-        <h4>Choose Region</h4>
-        <VDropdown
-          :options="regions"
-          display-field="name"
+        <h4>Selected Region</h4>
+        <VTextField
```

Comment:
Can we not keep the dropdown and disable it?

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/101/commits/b3554d5066efe946090a7ede6da4443ae41782b7